stable/weave-scope/charts/weave-scope-agent/templates/clusterrole.yaml

```diff
@@ -11,14 +11,61 @@ metadata:
     {{- include "weave-scope.annotations" . | indent 4 }}
 rules:
   - apiGroups:
-      - '*'
+      - ''
     resources:
-      - '*'
+      - pods
     verbs:
-      - '*'
-  - nonResourceURLs:
-      - '*'
+      - get
+      - list
+      - watch
+{{ if not .Values.serviceAccount.readonly }}
+      - delete
+{{ end }}
+  - apiGroups:
+      - ''
+    resources:
+      - pods/log
+      - services
+      - nodes
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - apps
+    resources:
+      - statefulsets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - batch
+    resources:
+      - cronjobs
+      - jobs
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+    resources:
+      - deployments
+      - daemonsets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+    resources:
+      - deployments/scale
```

Comment:
Are these permissions the ones officially supported by WeaveWorks?  Originally I took the permissions for the chart from the output of https://cloud.weave.works/k8s/scope.yaml?k8s-version=[version] (where [version] was 1.6.something].  If so, the annotations in the _helpers.tpl should be updated as well.

Reply:
Yes, these were official ones for version 1.9.dont_remember_now. Now that the latest version is 1.10, it's a good time to update the permissions once again.